Automattic/jetpack
# Mirroring: Link GitHub commit after push

#### Changes proposed in this Pull Request:

Link GitHub commit after push.

I was looking at the logs for a mirroring run and found myself wishing
they included a link to click to view the resulting commit.

#### Jetpack product discussion


None.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Run `.github/files/update-project-mirrors.sh` locally. Probably at least Jetpack will have a commit, see that a link is printed.

* Merge this.
* In the resulting run of the action, look for links in the logs. Click them and see that they go to the correct commit.

#### Proposed changelog entry for your changes:


* None needed.
